GowerMetric.py

```diff
@@ -154,16 +230,88 @@ def fit(self, X):
                     ratio_cols[:, zero_values_mask], axis=0
                 )
 
-            if self.ratio_scale_normalization == "kde":
-                n = X.shape[0]
-                c = 1.06
-
-                s = np.std(ratio_cols, axis=0)
-                self.h_ = (
-                    c / n ** (1 / 5) * np.min([s, self.ranges_ / 1.34], axis=0)
-                )
-
-        loader = GowerMetricWeights(self)
+            n = X.shape[0]
+
+            # h_t parameter
+            if self.ratio_scale_window == "kde":
+                if self.kde_type == "silverman":
+                    c = 1.06
+
+                    s = np.std(ratio_cols, axis=0)
+                    self.h_ = (
+                        c
+                        / n ** (1 / 5)
+                        * np.min([s, self.ranges_ / 1.34], axis=0)
+                    )
```

Comment:
I get that black formats this like this, but in those cases break code into variables, e.g.;
```
scaling = c / n ** (1 / 5)
bandwidth = np.min([s, self.ranges_ / 1.34], axis=0)
self.h_ = scaling * bandwidth
```